checker/checkFlags.ml

```diff
@@ -11,13 +11,19 @@
 open Declarations
 
 let set_local_flags flags env =
+  (* Explicitly ignored flags have been commented out *)
   let flags =
     { (Environ.typing_flags env) with
       check_guarded = flags.check_guarded;
       check_positive = flags.check_positive;
       check_universes = flags.check_universes;
       conv_oracle = flags.conv_oracle;
       share_reduction = flags.share_reduction;
+      enable_VM = flags.enable_VM;
```

Comment:
This will enable the vm regardless of the command line flag

Reply:
Reading the code, I was under the impression that make_senv was in charge of setting flags. This is to override specific flags no?

Reply:
Is enable_VM not set to true by default (in coqc)? Wouldn't that mean it was always enabled?

Reply:
Do we really want to be able to override this here anyway?

Reply:
Shouldn't this just be `envflags.enable_VM`?